### DISCUSSION TITLE
Is this letter recognized by MUFI?

### DISCUSSION THREAD

**Discussion by elechner** [psb1558/Junicode-font#89] [General]
**A**t:
![capitalA](https://user-images.githubusercontent.com/19876228/144450493-3d1d32c6-8ea0-4f20-8e14-2f22221da2d1.PNG)
**A**nna:
![Anna](https://user-images.githubusercontent.com/19876228/144458244-01bd7138-88f1-4363-840d-fbf1148838c2.PNG)
**A**lgꜵr:
![Algaor](https://user-images.githubusercontent.com/19876228/144462004-d6c245fb-7a95-4d2f-ae4d-684474f82585.PNG)

Source: 
https://handrit.is/en/manuscript/imaging/is/HolmPerg04-0015/31r-32r#page/31r++(1+of+3)/mode/2up
https://handrit.is/en/manuscript/imaging/is/HolmPerg04-0015#page/93v++(188+of+211)/mode/2up

Is this variant of capital A recognized by MUFI? 
It seems higher and larger than the recoginzed "LATIN ENLARGED LETTER SMALL A" which also is not a capital letter. Or what do you think? 
I would call this char a vairant of unrecognized "LATIN CAPITAL LETTER UNICAL A"/"LATIN CAPITAL LETTER A UNICAL FORM".

**Comment by psb1558** [comment]
I wasn't in on the original MUFI discussion about enlarged minuscules, so I'm not sure about all the thinking there, but I think the other MUFI people would take this as an enlarged minuscule. I wonder if the size in Andron Scriptor (the reference font for MUFI) was a compromise, knowing that these minuscules come in various sizes. MUFI has no cap uncial variant of A.

It occurs to me that in the variable version of Junicode I could make the enlarged minuscules max out at cap height. No need for compromise there—user can choose the size. Perhaps I'll experiment with a few letters.

**Reply by psb1558** [reply]
Following up, I'm thinking that if I made the "enlarged minuscules" cap height on the extreme end of the variable font's [Enlarge](https://psb1558.github.io/Junicode-font/EnlargedAxis.html) axis, that axis would be much more useful. One could set it to 0 for an lc-size character, 0.5 for a MUFI-size enlarged minuscule, 1.0 for the letter at cap height. An experiment:
<img width="415" alt="image" src="https://user-images.githubusercontent.com/16215767/144504527-712bf997-47a4-4e2f-b817-4de0de5af454.png">

**Comment by elechner** [comment]
MUFI writes this on _enlarged minuscules_: 
`Enlarged minuscules are recognised as separate characters by some scholars, cf. e.g. Andrea de Leeuw van Weenen (A Grammar of Möðruvallabók, CNWS 85, Leiden 2000). The traditional view has been to interpret these characters as variants of majuscules and encode them as such. It can be argued that this is a functional rather than a graphemic point of view and that it obscures the obvious distinction between e.g. ‘A’ (the majuscule) and ‘’ (the enlarged minuscule).`
Source: https://folk.uib.no/hnooh/mufi/specs/MUFI-CodeChart-4-0.pdf

Given this statement, shouldn't default size be the same as its standard variant? 
Looking forward to trying out full size enlarged minuscules.

**Comment by psb1558** [comment]
I know that passage, and I've never found it particularly clarifying. It seems to address more how to associate "enlarged minuscules" (i.e. with upper- or lowercase letters) than how big they should be. I don't want to get too out of step with MUFI here, at least in the static fonts. Anyway, intermediate-sized sentence-initial letters with lowercase shape actually are a thing; but I doubt that scribes made our binary distinction of lower- and uppercase.

Is there even a "standard variant" for these things?

But I like the possibilities for the variable axis very much.

**Reply by elechner** [reply]
> I know that passage, and I've never found it particularly clarifying. It seems to address more how to associate "enlarged minuscules" (i.e. with upper- or lowercase letters) than how big they should be. I don't want to get too out of step with MUFI here, at least in the static fonts. Anyway, intermediate-sized sentence-initial letters with lowercase shape actually are a thing; but I doubt that scribes made our binary distinction of lower- and uppercase.
> 
> Is there even a "standard variant" for these things?
> 
> But I like the possibilities for the variable axis very much.

Standard variant was bad wording on my side. I'd like to see default capital  'A' to have char variant enlarged minuscule 'a' added to cv01 etc, as described by MUFI: `The traditional view has been to interpret these characters as variants of majuscules and encode them as such. `

**Comment by psb1558** [comment]
Okay, this is very interesting. Do you mind if I think aloud for a moment?

This idea would make the enlarged minuscules similar (in programming approach) to small caps, which most fonts make variants of both caps (via c2sc) and lowercase letters (via smcp). For small caps you have two different features because typographers want to have a choice of styles in a run of text: all small cap (c2sc + smcp) and mixed cap and small cap (smcp alone).

But for this feature one doesn't have to think about runs of text: it's meant to be applied to isolated characters at the beginnings of sentences. And so I can't think of any reason not to have a cap-to-enlarged transformation on the same feature as lc-to-enlarged, i.e. ss06. Right now the enlarged minuscules are in the font twice: once on their MUFI/PUA code points (e.g. **U+EEE0** for LATIN ENLARGED LETTER SMALL A) and once unencoded and using Adobe's naming conventions (e.g. **a.enlarged**), which allow them to be searched on web pages and in PDFs. ss06 produces the unencoded version.

To make them **also** variants of caps, it would only be necessary to duplicate them once again, unencoded and with cap names (e.g. **A.enlarged**), and put them in ss06 with the others. So in case-sensitive searches they'd be treated as caps.

I can't think of any reason this shouldn't work, and I'd rather keep the enlarged minuscules out of the cvNN features, as I have done so far.

**Comment by elechner** [comment]
I'd like to be able to replace specific capital letters with its enlarged minuscule variant without having to replace all. 

Also there are two enlarged minuscule variants of Ꜵ in MUFI: 
1. LATIN LIGATURE ENLARGED LETTER SMALL A AND LATIN SMALL LETTER O
2. LATIN ENLARGED LETTER SMALL LIGATURE AO

Here is variant 1. in '**Ꜵ**ndveges' and '**ꜵ**ndveges' in same picture: 
![Aondveges](https://user-images.githubusercontent.com/19876228/144692012-194317ee-ddd3-4791-a6ff-ca2eee4dd8f8.png)

Source: https://handrit.is/en/manuscript/imaging/is/HolmPerg04-0015#page/88v++(178+of+211)/mode/2up

Also two enlarged minuscule variants of D, i think... Unicode and MUFI doesn't have a regular SMALL LETTER D ROTUNDA (without diacritics): 
1. LATIN ENLARGED LETTER SMALL D
2. LATIN ENLARGED LETTER D ROTUNDA

Enlarged minuscules at MUFI: https://mufi.info/m.php?p=mufichars&i=3&v=2

Style sets and character variants are not mutually exclusive. A style set like ss06, but for capital letters, might very well be of interest. But I think character variants are of more interest when replicating old texts as they might have a mix like enlarged minuscule a for A, regular capital B, and uncial E etc.

**Comment by psb1558** [comment]
Okay, that makes sense. Here's the plan, then:

1. Create uppercase enlarged letters (A.enlarged, etc.), visually identical to the lowercase ones.
2. Add the uppercase enlarged letters to ss06.
3. Add both upper- and lowercase enlarged letters to the appropriate cvNN features.

This may take a few days.

In addition, in the variable font, the size of the letters at the upper extreme of the Enlarge axis will be about the size of caps, and with about the same weight.

**Reply by psb1558** [reply]
It's been a while, but I'll mention anyway that this is all done now. The current setup provides various ways to get these variants, each with its particular advantages and disadvantages.
